### PR TITLE
Make track section navigation buttons holdable

### PR DESCRIPTION
**Pull request by mrmagic2020**
Implements #22171.

### CODE REVIEW

**Review by ZeeMaji — APPROVED** [review]
Gave it some testing on most of the tracked rides from magic mountain, it seems to be working as expected.

**Review by AaronVanGeffen — APPROVED** [review]
Code changes look good to me!

File: distribution/changelog.txt [hunk 1]
```diff
@@ -28,6 +28,7 @@
 - Change: [#21214] Wacky Worlds and Time Twister’s scenario names now match their park names.
 - Change: [#21991] UI themes JSON now use colour names and a translucency bool, instead of a number (old themes still work).
 - Change: [#22057] Reorder Time Twister’s scenarios and adjust their difficulty classification.
+- Change: [#22196] Make track nagivation buttons holdable.
```
Comment (mrmbernardi) [inline distribution/changelog.txt:31]: There's a typo in the word navigation.

Comment (mrmagic2020) [reply distribution/changelog.txt:31]: Oops, my bad. I'll open a PR to amend it.

**Pull request merged**